index.js

```diff
@@ -18,87 +17,113 @@ function Database (read_config, write_config = false) {
 
   if (write_config && read_config['host'] !== write_config['host']) {
     this.OUT = new Host(write_config);
-  }
-  else {
+  } else {
     this.OUT = this.IN;
   }
 }
 
 function Host (config) {
-  this.error_retries = 0;
+  this.connect_retries = 0;
+  this.query_retries = 0;
   this.error_timeout = 10000;
   this.config = config;
-  connect(this);
+  this.config.reconnect = false;
```

Comment:
how/where is this being used?

Reply:
config.reconnect is used by the package promise-mysql. I set it to false so the lib doesn't try to reconnect by itself since this is taken care of by our error event handler

Reply:
Aha, OK.. we'll probably just use the reconnect from the library in the future.. its ok for now